Matrix.py

```diff
@@ -0,0 +1,120 @@
+        matrix_to_return = Matrix((tuple(sub_list)))
+        return matrix_to_return
+
+    # ones method returns a Matrix full of ones which sizes determined by "unit"
+    @staticmethod
+    def ones(unit):
+        major_list = [([1] * unit) for i in range(unit)]
+        sub_list = [tuple(lt) for lt in major_list]  # convert list of lists to type Matrix. lt are the inside lists
+        matrix_to_return = Matrix((tuple(sub_list)))
+        return matrix_to_return
+
+    def __add__(self, other):
+        if type(other) is Matrix:
+            ans = []
+            for tup in range(len(self.data)):
+                ans.append([])
+                for num in range(len(self.data)):
+                    ans[tup].append(self.data[tup][num] + other[tup][num])
+            sub_list = [tuple(ls) for ls in ans]  # convert ans to type Matrix. ls are the inside lists
+            matrix_to_return = Matrix((tuple(sub_list)))
+            return matrix_to_return
+        else:
+            raise SyntaxError("only matrix's summery")
+
+    def __sub__(self, other):
+        if type(other) is Matrix:
+            ans = []
+            for tup in range(len(self.data)):
+                ans.append([])
+                for num in range(len(self.data)):
+                    ans[tup].append(self.data[tup][num] - other[tup][num])
+            sub_list = [tuple(ls) for ls in ans]  # convert list of lists to type Matrix. ls are the inside lists
+            matrix_to_return = Matrix((tuple(sub_list)))
+            return matrix_to_return
+        else:
+            raise SyntaxError("only matrix's subscription")
+
+    def __truediv__(self, other):
+        if type(other) is float or int:  # the program doesnt support division by other Matrix
+            return self.scalar_mul(1/other)
+        else:
+            raise SyntaxError("only  scalar division")
+
+    def __mul__(self, other):
+        if type(other) is Matrix:  # Matrix's multiplication --->
+            ans = []  # will contain the answer as a list of lists.
+            new_other = []
+            # this loop convert "other" from Matrix type to a list of lists named "new_other".
+            for line in range(len(self.data)):
+                new_other.append([])
+                ans.append([0] * len(self.data))  # ns starts as a list of lists full of zeros.
+                for num in range(len(self.data)):
+                    new_other[line].append(other[num][line])
+            # this is the multiplication process->
+            for tup in range(len(self.data)):
+                for num in range(len(self.data)):
+                    for i in range(len(self.data)):
+                        ans[tup][num] += self.data[tup][i] * new_other[num][i]
+            # here i added to the zero the summery of the multiplications of any two numbers which
+            # their places in the matrices are fits by the matrices multiplication rules.
```

Comment:
when you document and explain your code, try to avoid using "i" in your explanations.